webservice/soap/include/inc.soap_functions.php

```diff
@@ -63,6 +63,14 @@ public static function  loginLDAP($client, $username, $password)
 		return $sua->loginLDAP($client, $username, $password);
 	}
 
+	public static function  loginStudipUser($sid,$user_id)
```

Comment:
According to the ILIAS Jour Fixe: Please mark this method/service as @deprecated and add a short note, because this service will be removed with ILIAS 5.3.x

- http://www.ilias.de/mantis/view.php?id=18992